general_itests/validate.feature

```diff
@@ -0,0 +1,13 @@
+Feature: paasta validate can be used
+
+  Scenario: Running paasta validate against a valid service
+    Given a "valid" service
+    When we run paasta validate
+    Then it should have a return code of "0"
+     And everything should pass
+
+  Scenario: Running paasta validate against an invalid service
+    Given an "invalid" service
+    When we run paasta validate
+    Then it should have a return code of "1"
+     And something should fail
```

Comment:
I wasn't sure what this meant, can you clarify the string, how about:
"It should report the error in the output"